examples/basic_mysql_connection/Jalal_DataBase.sql

```diff
@@ -0,0 +1,32 @@
+
+create table jalal_doctors(
```

Comment:
Dear Jalal It is better to use capital letters Mysql reserved worlds such as CREATE TABLE, and small letters to variables names.
and it is better uo use IF NOT EXISTS when crating a table to avoid any error.